lonestardist/CMakeLists.txt

```diff
@@ -99,15 +99,23 @@ function(add_test_dist_and_verify app input type part N np)
 
 
   # Don't run dist tests in CI since they are broken in too many cases right now.
```

Comment:
Please remove this comment as well.

Reply:
Done locally; will push once everyone has done a review to avoid pushing small changes like this.